templates/full-time-program.html

```diff
@@ -51,17 +49,39 @@ <h1 class="blue-h1">The Program</h1>
       <div class="blue-background">
         <canvas id="myChart"></canvas>
       </div>
+      <p class="justify">
+        As many companies have changed their priorities over the last few years, Techtonica 
+        has had difficulty relying on a corporate sponsorship model. As a result, Techtonica 
+        is adopting a tuition-based model with need-based tuition and stipend 
```

Comment:
"with need-based tuition" —> "with need-based, sliding-scale tuition"

Reply:
### ✅ Addressed in [7139de1](https://github.com/Techtonica/techtonica.org/pull/336/commits/7139de1caac514b497b04a2cfc08ed0517b0da00).